### ISSUE TITLE
Using the DNSCrypt proxy releases 2.0.45

### ISSUE THREAD

**Issue by sutex** [bitbeans/SimpleDnsCrypt#529]
Is it as simple as replacing the older SimpleDnsCrypt dnscrypt-proxy, with the newer 2.0.45?

https://github.com/DNSCrypt/dnscrypt-proxy/releases

**Comment by AdKiller** [comment]
I was wondering the same, but I think it is unlikely...
If I have spare time and energy, I will test it on VM

**Comment by sutex** [comment]
Yeah the Dev seems to have disappeared nothing happening here for quite sometime

**Comment by Mongogamer** [comment]
As long as there are no breaking changes introduced by dnscrypt-proxy you can simply replace the included version with the latest release from Github. I have done this myself before when the update cycle was far behind and it worked without issues.

**Comment by sutex** [comment]
Well there seems to be _breaking changes_ ? As the blacklists have been changed

**Comment by Mongogamer** [comment]
After investigating the source it seems like the blacklist format is still supported in 2.0.45. It will however be replaced in the future when version 2.1.x releases, which would require an update of SimpleDNScrypt. For now there is a fallback mechanic for old configs in place so it should be safe to upgrade.

**Comment by sutex** [comment]
April it will be 12 months since the last update. And the Dev seems to have disappeared very quite, seen no comments. Which brings its own drama as GUI devotee, soon if no updates, will have to migrate to the command line.

**Comment by AdKiller** [comment]
> April it will be 12 months since the last update. And the Dev seems to have disappeared very quite, seen no comments. Which brings its own drama as GUI devotee, soon if no updates, will have to migrate to the command line, versions?

I guess it is time to roll up your sleeves and install latest dnscrypt from command line. It doesn't seem too much of a headache... 

https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Installation-Windows

**Comment by bitbeans** [comment]
I have more time again and am working on an update. Sorry for the delay!

**Comment by rmcohen3** [comment]
> I have more time again and am working on an update. Sorry for the delay!

Welcome back! Looking forward to your update.

**Comment by rescenic** [comment]
How to Update dnscrypt-proxy to version 2.0.45:
https://github.com/bitbeans/SimpleDnsCrypt/issues/537#issuecomment-817123129

**Comment by sutex** [comment]
@bitbeans Any update on the coming update?

**Comment by sutex** [comment]
It seems you can no longer update SimpleDnsCrypt  dnscrypt-proxy? with https://github.com/DNSCrypt/dnscrypt-proxy/releases

https://github.com/bitbeans/SimpleDnsCrypt/issues/559
Abandoned project - A List of Alternatives? #559

**Comment by Dola-Shuvi** [comment]
I have created an [unofficial 0.7.2 version](https://github.com/Dola-Shuvi/SimpleDnsCrypt/releases/tag/0.7.2) for now in case anyone doesn't want to compile the code I created in #558 themselves. Please read the release notes carefully to prevent avoidable troubleshooting.
